src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -15,8 +15,6 @@ def serialize(self, obj, attrs, user):
             "id": six.text_type(obj.id),
             "name": obj.name,
             "organizationId": six.text_type(obj.organization_id),
-            # TODO: Remove this once we've migrated to org level
-            "projectId": six.text_type(obj.query_subscriptions.first().project_id),
```

Comment:
@wedamija is this ok to remove?

Reply:
Yeah it's fine with me, I assume it'll break the project level pages, but that shouldn't matter too much.

Reply:
The project level pages are being removed in this PR so shouldn't be a problem